ethereum/forge-test/Pyth.t.sol

```diff
@@ -0,0 +1,444 @@
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+import "forge-std/Test.sol";
+
+import "@pythnetwork/pyth-sdk-solidity/IPyth.sol";
+import "@pythnetwork/pyth-sdk-solidity/PythStructs.sol";
+import "./utils/WormholeTestUtils.t.sol";
+import "./utils/PythTestUtils.t.sol";
+import "./utils/RandTestUtils.t.sol";
+
+contract PythTest is Test, WormholeTestUtils, PythTestUtils, RandTestUtils {
+    IPyth public pyth;
+
+    // -1 is equal to 0x111111 which is the biggest uint if converted back
+    uint64 constant MAX_UINT64 = uint64(int64(-1));
+    
+    function setUp() public {
+        pyth = IPyth(setUpPyth(setUpWormhole(1)));
+    }
+
+    function generateRandomPriceAttestations(
```

Comment:
This method and the method below could be moved to the PythTestUtils but I won't move them now. I will move them if another test wants to use them (which will probably mean dividing Pyth tests into multiple files).